PyInstaller/depend/bindepend.py

```diff
@@ -125,8 +125,7 @@ def _getImports_pe(pth):
                 # sym.forwarder is a bytes object. Convert it to a string.
                 forwarder = winutils.convert_dll_name_to_str(sym.forwarder)
                 # sym.forwarder is for example 'KERNEL32.EnterCriticalSection'
-                dll, _ = forwarder.split('.')
-                dlls.add(dll + ".dll")
+                dlls.add(forwarder.split('.')[0] + ".dll")
```

Comment:
This looks like to me. You might update the comment to give the failing case ("KERNEL32.EnterCriticalSection.SomethingElseHere").

Reply:
I suggest only changing the first line (and of corure update the comment as bjones1 suggested.
```suggestion
                dll  = forwarder.split('.')[0]
                dlls.add(dll + ".dll")
```